webservice/assets/peoples.js

```diff
@@ -124,5 +124,17 @@ export function getPeopleList() {
 |         2022 Academician at Daskom1337         |
 +------------------------------------------------+`,
     },
+    {
+      Avatar: "icen.html",
+      Handle: "vincentiusantu",
+      Realname: "Vincentius Adisurya Fransisco Antu",
+      Description: `
+      +------------------------------------------------+
+      |         I'm confused what to fill so           |
+      |      put anything here, anything really.       |
+      |                      ~~~                       |
+      |         2022 Academician at Daskom1337         |
+      +------------------------------------------------+`,
+    }
```

Comment:
```suggestion
      Description: `
+-------------------------------------------------+
|         I'm confused what to fill so            |
|      put anything here, anything really.        |
|                      ~~~                        |
|         2022 Academician at Daskom1337          |
+-------------------------------------------------+`,
    }
```

This one should be changed like this first @vincentiusantu, it wont look correct in the website if it doesnt follow the same box as the other description, change this first, commit and push it and then re-request my review by clicking on the top right corner button in the same place you add me as reviewer before, thanks vincent 👍

Reply:
Done bang